src/api/v1/projects/project_routes.py

```
@@ -11,6 +11,41 @@
 project_router = APIRouter(tags=["projects"])
 
 
+@project_router.post(
+    "/create/new", status_code=status.HTTP_201_CREATED,
+    response_model=uuid.UUID
+)
+async def create_project(
+    project: CreateProject,
+    user: User = Depends(current_active_user),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)